lib/puppet/type/java_ks.rb

```diff
@@ -149,6 +149,10 @@ def value=(*values)
     defaultto 120
   end
 
+  newparam(:source_password) do
+    desc "The source keystore password"
```

Comment:
Is this only valid when `storetype => 'pkcs12'`? If so, it should do some validation as such.